README.md

```diff
@@ -1543,3 +1543,24 @@ openstackbackuprestore   restore    openstackbackupsave-1641928378   Restored
 ```
 
 At this point, all resources contained with the chosen `OpenStackBackup` should be restored and fully provisioned.
+
+# Day2 Operations
+
+## Change resources on virtual machines
+
+If required it is possible to change CPU/RAM of an openstackvmset configured via the openstackcontrolplane. The workflow is as follows:
+
+* change/patch the virtualMachineRole within the virtualMachineRoles list of the openstackcontrolplane CR
+
+E.g. to change the controller virtualMachineRole to have 8 cores and 21GB of RAM:
+
+```bash
+oc patch -n openstack osctlplane overcloud --type='json' -p='[{"op": "add", "path": "/spec/virtualMachineRoles/controller/cores", "value": 8 }]'
+oc patch -n openstack osctlplane overcloud --type='json' -p='[{"op": "add", "path": "/spec/virtualMachineRoles/controller/memory", "value": 21 }]'
+
+oc get osvmset
+NAME         CORES   RAM   DESIRED   READY   STATUS        REASON
+controller   8       21    1         1       Provisioned   All requested VirtualMachines have been provisioned
+```
+
+* schedule a restart of the virtual machine to get the change reflected inside the virtual machine (**Important** it is required to power off/on the virtual machine)
```

Comment:
Should we mention something about `virtctl restart <VM>` here in regards to this requirement?

Reply:
I had a broken file system when using virtctl restart. therefore graceful shutdown from inside the VM and virtctl start is probable better

Reply:
virtctl restart should be graceful, are we missing a guest agent on the VM?

Reply:
no its there, but I think its because we set terminationGracePeriodSeconds: 0 so that when there is a fencing action triggered they get immediately switched off.